packages/components/src/templates/next/layouts/IndexPage/IndexPage.tsx

```diff
@@ -54,14 +56,20 @@ const IndexPageLayout = ({
             layout,
             site,
             LinkComponent,
+            permalink: page.permalink,
           })}
 
-          <ChildrenPages
-            permalink={page.permalink}
-            site={site}
-            LinkComponent={LinkComponent}
-            {...childpages}
-          />
+          {!hasChildpageBlock && (
+            <ChildrenPages
+              permalink={page.permalink}
+              site={site}
+              LinkComponent={LinkComponent}
+              variant={CHILDPAGE_LAYOUT_OPTIONS.Rows}
+              showSummary={true}
+              showThumbnail={false}
+              type="childpage"
+            />
+          )}
```

Comment:
nit: leave it up to you on this one but thought would be cleaner if we updated `content` directly with the `childpages` block if one doesn't exist yet, so we skip this direct call to the `ChildrenPages` block from this layout.

Reply:
ok, that's doable tooj1